lib/omniauth/strategies/openid_connect.rb

```diff
@@ -57,6 +57,14 @@ class OpenIDConnect
       option :extra_authorize_params, {}
       option :allow_authorize_params, []
       option :uid_field, 'sub'
+      option :pkce, false
+      option :pkce_verifier, nil
+      option :pkce_options, {
+        code_challenge: proc { |verifier|
+          Base64.urlsafe_encode64(Digest::SHA2.digest(verifier), padding: false)
```

Comment:
original discussion: https://github.com/omniauth/omniauth_openid_connect/pull/80/files#r802033038

This hasn't been addressed, because these digests are not interchangeable and the feature does not work with just Digest::SHA2.hexdigest(verifier).

Reply:
Thanks for checking that.